scripts/minio-init.sh

```
@@ -0,0 +1,74 @@
+#!/usr/bin/env sh
+set -euo pipefail 2>/dev/null || set -euo
```

Shellcheck (0.10.0)

[warning] In POSIX sh, set option pipefail is undefined.

(SC3040)